backend/backend.go

```diff
@@ -295,6 +300,14 @@ func GetKeyHierarchy(vfs afero.Fs, state *config.State) (*KeyHierarchy, error) {
 }
 
 func GetBackendType(b []byte) (BackendType, error) {
+	if json.Valid(b) {
+		var yubiData YubikeyData
+		if err := json.Unmarshal(b, &yubiData); err != nil {
```

Comment:
You can probably just omit the variable assignment if the intention is to just check if the format serializes into `YubikeyData`.

> if err := json.Unmarshal(b, &YbuikeyData{}); err != nil {